neware_parser/PlotEngine.py

```diff
@@ -0,0 +1,120 @@
+import pickle
+
+import matplotlib.patches as mpatches
+
+# TODO(harvey) duplicate in plot.py
+COLORS = [
+    (.4, .4, .4),
+
+    (1., 0., 0.),
+    (0., 0., 1.),
+    (0., 1., 0.),
+
+    (.6, 0., .6),
+    (0., .6, .6),
+    (.6, .6, 0.),
+
+    (1., 0., .5),
+    (.5, 0., 1.),
+    (0., 1., .5),
+    (0., .5, 1.),
+    (1., .5, 0.),
+    (.5, 1., 0.),
+]
+
+
+class PlotEngine:
```

Comment:
I don't intend to keep the classes in the final product. It's easier for me during dev to keep them separate, since I don't have a generic plotting function yet, and that's the end goal